main.go

```diff
@@ -290,6 +290,7 @@ type payload struct {
 	StreamStart     string `config:"streamStart"`
 	StreamEnd       string `config:"streamEnd"`
 	TargetTimezone  string `config:"timezone"`
+	SkipLoad        bool   `config:"skip-load"`
```

Comment:
Nit: Should we make this skipLoad to match the case of streamStart/streamEnd? Though I don't what our standard is, seems like it's all over the place

Reply:
(I don't feel strongly about this)

Reply:
I guess `skip-debounce` also has a dash but we didn't have two word config params in the other repros so I didn't think about it then.

Reply:
Chatted offline. Switched it to be camelCase, and consistent across the repos with skipDebounce.